src/middleware/__tests__/addUserContext.int.test.ts

```diff
@@ -9,15 +9,21 @@ describe('requireAuthentication', () => {
 		const mockRequest = {} as unknown as AuthenticatedRequest;
 		const mockResponse = {} as unknown as Response;
 
-		const runAssertions = async (err: unknown) => {
-			expect(err).toBe(undefined);
-			const metrics = await loadTableMetrics('users');
-			expect(metrics.count).toEqual(0);
-			expect(mockRequest.user).toBe(undefined);
-		};
+		loadTableMetrics('users')
```

Comment:
To clarify, we are refactoring the tests to be inside the loadTableMetrics scope because we are now interested in the value returned(baselineUserCount)? Because now we want to know that the count value in the users table is equal to ... the count value in the users table(?)

Reply:
Yeah that's right -- we want to be sure the value did not change, and to do that we need to know what the value was in the first place.

Normally we could have an async test and await this (avoid the next / catch) but since middleware is weird and uses callbacks instead of promises we need to be able to use jest's `done` -- and that doesn't exist in an async context.